libs/traits/src/liquidity_pools.rs

```diff
@@ -30,6 +30,10 @@ pub mod test_util {
 	use super::*;
 
 	#[derive(Debug, Eq, PartialEq, Clone, Encode, Decode, TypeInfo, MaxEncodedLen)]
+	#[cfg_attr(
+		any(test, feature = "std", feature = "runtime-benchmarks"),
+		derive(Default)
+	)]
```

Comment:
NIT. Because this type is never used for production, it's ok to add `Default` normally